csm/core/controllers/login.py

```diff
@@ -48,11 +48,11 @@ async def post(self):
         password = body.get('password', None)
 
         session_id, body = await self.request.app.login_service.login(username, password)
-        Log.debug(f"Obtained session id for {username}")
+        Log.info(f"Obtained session id for {username}")
```

Comment:
for this log statement, debug would be more appropriate log level.
successful login msg is already set to info ( line 55).

Reply:
this log statement is not required. Remove it